frontend/src/screens/dashboard/docsqa/main/partials/option.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+import { MenuItem, Select } from '@mui/material'
+
+interface OptionProps {
+  title: string
+  placeholder: string
+  initialValue: string
+  data: any[] | undefined
+  handleOnChange: (e: any) => void
+}
+
+const Option = (props: OptionProps) => {
```

Comment:
let's call it **ConfigSelector** if we want to create this component dedicatedly for configs select dropdown in main page ... but if we want to make it more generic then we have to refactor this component

Reply:
I hade made an atomic picker component, its actually there in my webcrawler branch https://github.com/truefoundry/cognita/blob/feat/web-crawler/frontend/src/components/base/atoms/Picker.tsx